deployability/modules/jobflow/templates/manager/Test-manager-complete.yaml

```diff
@@ -62,21 +63,23 @@ tasks:
             - wazuh-1: "{working-dir}/manager-linux-centos-7-amd64/inventory.yaml"
             - wazuh-2: "{working-dir}/manager-linux-ubuntu-20.04-amd64/inventory.yaml"
             - wazuh-3: "{working-dir}/manager-linux-ubuntu-22.04-amd64/inventory.yaml"
-            - wazuh-4: "{working-dir}/manager-linux-redhat-7-amd64/inventory.yaml"
-            - wazuh-5: "{working-dir}/manager-linux-redhat-8-amd64/inventory.yaml"
-            - wazuh-6: "{working-dir}/manager-linux-redhat-9-amd64/inventory.yaml"
-            - wazuh-7: "{working-dir}/manager-linux-centos-8-amd64/inventory.yaml"
-            - wazuh-8: "{working-dir}/manager-linux-debian-10-amd64/inventory.yaml"
-            - wazuh-9: "{working-dir}/manager-linux-debian-11-amd64/inventory.yaml"
-            - wazuh-10: "{working-dir}/manager-linux-debian-12-amd64/inventory.yaml"
-            - wazuh-11: "{working-dir}/manager-linux-amazon-2-amd64/inventory.yaml"
+            - wazuh-4: "{working-dir}/manager-linux-ubuntu-24.04-amd64/inventory.yaml"
+            - wazuh-5: "{working-dir}/manager-linux-redhat-7-amd64/inventory.yaml"
+            - wazuh-6: "{working-dir}/manager-linux-redhat-8-amd64/inventory.yaml"
+            - wazuh-7: "{working-dir}/manager-linux-redhat-9-amd64/inventory.yaml"
+            - wazuh-8: "{working-dir}/manager-linux-centos-8-amd64/inventory.yaml"
+            - wazuh-9: "{working-dir}/manager-linux-debian-10-amd64/inventory.yaml"
+            - wazuh-10: "{working-dir}/manager-linux-debian-11-amd64/inventory.yaml"
+            - wazuh-11: "{working-dir}/manager-linux-debian-12-amd64/inventory.yaml"
+            - wazuh-12: "{working-dir}/manager-linux-amazon-2-amd64/inventory.yaml"
           - tests: "install,restart,stop,uninstall"
           - component: "manager"
           - wazuh-version: "<major.minor.patch>"
           - wazuh-revision: "<revision>"
           - live: False
     depends-on:
       - "allocate-manager-linux-ubuntu-20.04-amd64"
+      - "allocate-manager-linux-ubuntu-24.04-amd64"
       - "allocate-manager-linux-ubuntu-22.04-amd64"
```

Comment:
Keep the same order as other lists

- Ubuntu 20
- Ubuntu 22
- Ubuntu 24

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5689/commits/1be313b0a617de7de07bf2db470eb18b0219916a